resource/interface/client/context.lua

```diff
@@ -115,7 +117,36 @@ RegisterNUICallback('clickContext', function(id, cb)
     })
 end)
 
-RegisterNUICallback('closeContext', closeContext)
+RegisterNUICallback('onEnter', function(id, cb)
+    cb(1)
 
+    if math.type(tonumber(id)) == 'float' then
+        id = math.tointeger(id)
+    elseif tonumber(id) then
+        id += 1
+    end
```

Comment:
This conversion can go into it's own function I think, to avoid repeating it 3 times.